### ISSUE TITLE
Flesh out kp

### ISSUE THREAD

**Issue by busstoptaktik** [busstoptaktik/geodesy#8]
Compact cmd line notation, actual functionality, help text

**Comment by busstoptaktik** [comment]
```sh
kp "cart ellps: intl | helmert dx: -87 dy: -96 dz: -120 | cart inv ellps: GRS80" foo bar baz
```

by trivial substitutions this is transformed to

```yaml
pipeline: {
    steps: [
        cart: {ellps: intl},
        helmert: {dx: -87, dy: -96, dz: -120},
        cart: {inv: true, ellps: GRS80}
    ]
}
```

**Comment by busstoptaktik** [comment]
Closed with e8c3250d40d3bfb86732d85a784fa6afd859825f
